worlds/generic/docs/advanced_settings_en.md

```diff
@@ -174,6 +174,8 @@ A Link to the Past:
     - Moon Pearl
   start_location_hints:
     - Spike Cave
+  priority_locations:
+    - Links House
```

Comment:
Heads up, the location name is `Link's House` in the data package, so this would be ignored if they copied this verbatim. :P

<img width="190" alt="image" src="https://user-images.githubusercontent.com/11338376/176479705-0706ca4e-5068-46d4-838c-c92034fbb817.png">

Reply:
hate.